src/Infrastructure/Migrations/Version20230302085340.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Infrastructure\Migrations;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230302085340 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Populates the sylius_price_history_channel_pricing_log_entry table with initial data';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->abortIf(
+            !$this->getSchemaManager($this->connection)->tablesExist(['sylius_price_history_channel_pricing_log_entry']),
+            'The table sylius_price_history_channel_pricing_log_entry does not exist.',
+        );
+
+        $this->addSql(<<<SQL
+            INSERT INTO `sylius_price_history_channel_pricing_log_entry` (`channel_pricing_id`, `price`, `original_price`, `logged_at`)
+            SELECT `id`, `price`, `original_price`, NOW() FROM `sylius_channel_pricing`
+        SQL);
+    }
+
+    public function down(Schema $schema): void
+    {
```

Comment:
I wonder if we should not clean this table here, but it could be risky to delete not only what we want, even if it's the initiation of the plugin and this table. In any case, this problem will not exist if we move this logic to the migration, where we create this table

Reply:
Not needed anymore, as after moving this SQL to the creating one the table is dropped on the down.